articles/container-registry/container-registry-faq.md

```diff
@@ -407,6 +408,17 @@ curl $redirect_url
 
 If you are using the Microsoft Edge/IE browser, you can see at most 100 repositories or tags. If your registry has more than 100 repositories or tags, we recommend that you use either the Firefox or Chrome browser to list them all.
 
+### Why is Azure portal unable to send request for fetching repositories or tags?
+
+The browser might be unable to send the request for fetching repositories or tags to the server. There could be various reasons, including but not limited to:
+
+* Lack of network connectivity
+* Firewall
+* Ad blockers
+* DNS errors
+
+Please contact your network administrator or check your network configuration and connectivity.
```

Comment:
You can also try an incognito/private session to avoid any stale browser cache/cookies.

Reply:
// cc @djyou